app/jobs/write_chapter_summary_job.rb

```
@@ -0,0 +1,67 @@
+class WriteChapterSummaryJob < MetaJob
+  self.max_retries = 3
+  self.openai_model = ENV['OPENAI_MODEL']
+
+  def perform(chapter_id)
+    @component = Chapter.find(chapter_id)
+    @book = @component.book
+    
```

RuboCop (1.69.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)